js/libs.js

```diff
@@ -153,6 +164,12 @@ Fliplet.Registry.set('fliplet-widget-notifications:1.0:core', function (data) {
     setTimer(DELAY - diff);
   }
 
+  function attachObservers() {
+    Fliplet.Hooks.on('pushNotificationReceived', function () {
```

Comment:
@tonytlwu rename to `pushNotification` as per https://github.com/Fliplet/fliplet-widget-push-notifications/blob/master/js/notifications.js#L183